data/templates/new_game_template

```diff
@@ -29,7 +34,10 @@ addons=
 # Player tribe (empty means random tribe). Default: Map's tribe.
 tribe=
 
-# The filename of the starting condition script (without path), or the internal name of the add-on defining the starting condition. Default: Headquarters.
+# The filename of the starting condition script (without path), or the internal name of the add-on defining the starting condition.
+# Default: Headquarters. Examples:
+# init="village.lua"                    # Script at: data/tribes/initialization/frisians/starting_conditions/village.lua
```

Comment:
I think this does not work, because each tribe has his own starting condition. How does the code knows to use 
`data/tribes/initialization/frisians/starting_conditions/village.lua`
or
`data/tribes/initialization/empire/starting_conditions/village.lua`

Maybe combine it with option `tribe=`?

Reply:
For each tribe, the full path of each starting condition is stated in `data/tribes/initialization/<tribename>/init.lua`. The algorithm picks the first starting condition script whose filename matches the given value. 
This ensures that you can only choose a starting condition that is actually available for the set tribe (anything else would give an error).

Reply:
Sorry for the noise, i have made a failure by commenting the line `[player_1]` ... :-(